File: src/main/java/ru/tidstu/testingsystem/domain/TestData.java
package ru.tidstu.testingsystem.domain;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
import lombok.extern.log4j.Log4j;

@Log4j
@Getter
@Setter
@ToString
public class TestData {

    String inputData;
    String outputData;
    String nameQuestion;

    @Builder
    public TestData(String inputData, String outputData, String nameQuestion){
        this.inputData = inputData;
        this.outputData = outputData;
        this.nameQuestion = nameQuestion;
    }

}

File: ProgChecker/Sources/Data/Entity/TestData.cpp
#include "TestData.h"

void TestData::setInputData(const std::string &inputData)
{
    _inputData = inputData;
}

std::string TestData::getInputData() const
{
    return _inputData;
}

void TestData::setOutputData(const std::string &outputData)
{
    _outputData = outputData;
}

std::string TestData::getOutputData() const
{
    return _outputData;
}

std::string TestData::toString() const
{
    return "OutputData: [" + _outputData + "] InputData: [" + _inputData + "]\n";
}